webpack.config.js

```diff
@@ -93,6 +93,19 @@ module.exports = [
         },
       ],
     },
+
+    resolve: {
+      alias: {
+        app: path.resolve(__dirname, './client/control/app.js'),
+        routes: path.resolve(__dirname, './client/control/routes.js'),
+        stores: path.resolve(__dirname, './client/control/stores'),
+        reducers: path.resolve(__dirname, './client/control/reducers'),
+        actions: path.resolve(__dirname, './client/control/actions'),
+        components: path.resolve(__dirname, './client/control/components'),
+        tests: path.resolve(__dirname, './client/control/tests'),
+        selfrepair: path.resolve(__dirname, './client/selfrepair'),
```

Comment:
I quite like the idea of having these sort of absolute imports. I'm concerned about the predictability of these though. Since they are all at varying depths, it is hard to tell what any given import will actually point to.

I think we should go with a simpler, more predictable model, something like

```js
resolve: {
  alias: {
    actions: path.resolve(__dirname, './client/actions'),
    control: path.resolve(__dirname, './client/control'),
    selfrepair: path.resolve(__dirname, './client/selfrepair'),
    tests: path.resolve(__dirname, './client/tests'),
  },
}
```